fastlane/metadata/android/de/full_description.txt

```diff
@@ -0,0 +1,14 @@
+Deine Medien, zu Deinen Bedingungen.
```

Comment:
I wondered whether we should really capitalize the "du/dein*" mid-sentence. It's definitely more polite, but it also feels a bit outdated and kinda out-of-place on the internet (in contrast to e.g. a letter).

Reply:
Ugh, am I that old already? :scream: It's your repo, so of course it's your choice. We can also leave it as-is for now – and you can change it later if you think it should be changed? Else let me know, and I make another commit.

Reply:
Well, it's fine as is, so we can also keep it like that.